pymongo/event_listeners.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2020-present MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from pymongo import monitoring
+
+
+class CommandListener(monitoring.CommandListener):
+    """A simple example command listener
```

Comment:
Let's make these docstrings a bit more descriptive and not mention 'example' in their descriptions. While the source code for these may serve as an example/guide for the implementation of more complex listeners, these listeners are intended to themselves be imported and directly used in user-code.

I suggest using docstring like:

```
A simple listener that logs command events.

Listens for :class:`~pymongo.monitoring.CommandStartedEvent`, :class:`~pymongo.monitoring.CommandSucceededEvent` and :class:`~pymongo.monitoring.CommandFailedEvent` events and logs them at the `INFO` severity level using :mod:`logging`.

.. versionadded:: 3.11
```

Note that we use this structure for most docstings - a short, one-line docstring followed by a more in-depth one.